MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include src/pydata_sphinx_theme/locale *.pot *.po *.mo
```

Comment:
My knowledge of Python packaging is limited. I think this change is needed to make the translation files part of the source distribution. However, this is a tad confusing because the pyproject.toml file already seems to be attempting to cover this, although I think it's incorrectly configured:

https://github.com/pydata/pydata-sphinx-theme/blob/08f2b78e4ba175b5017499e4e688a8d14923a09f/pyproject.toml#L7-L16

Reply:
Ah I added this when debugging what was happening and forgot to remove the file, will delete it as it is not needed